arch/arm/core/aarch32/cortex_m/tz/secure_entry_functions.ld

```diff
@@ -7,14 +7,21 @@
 #if CONFIG_ARM_NSC_REGION_BASE_ADDRESS != 0
 	#define NSC_ALIGN . = ABSOLUTE(CONFIG_ARM_NSC_REGION_BASE_ADDRESS)
 #elif defined(CONFIG_CPU_HAS_NRF_IDAU)
-	/* The nRF9160 needs the NSC region to be at the end of a 32 kB region. */
-	#define NSC_ALIGN . = ALIGN(0x8000) - (1 << LOG2CEIL(__sg_size))
+#ifdef CONFIG_SOC_NRF9160
+#define REGION_SIZE 0x8000
```

Comment:
What about using (32*1024) instead of 0x8000 so that its easier to see that it is 32kB? Some elsewher, but if hex values are used otherwise in this file I understand if you want to keep it consistent.

Reply:
I created a kconfig for this.